frontend/components/modal/MissingTranslations.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+const { $axios, $missingTranslations } = useNuxtApp()
+const { missingTranslations: { missingTranslationsVisible } } = useModal()
+const { t } = useI18n()
+
+const prefix = 'components/modal/MissingTranslations'
+const existing = useState<Map<string, string>>(`${prefix}.existing`, () => new Map())
+const languages = ['en', 'es', 'fr', 'de']
+const tt = (key: string) => t(`${prefix}.${key}`)
+
+onMounted(async () => {
+  for (const lang of languages) {
+    const langData = await $axios({
+      method: 'get',
+      url: `/_nuxt/lang/${lang}.json`,
+    }).then((r) => {
+      return JSON.stringify(r.data)
```

Comment:
nit: Looks like the data is being parsed into JSON just to be turned back to a string again (and then, turned back into JSON below!), which is wasteful (esp for larger responses). You can [disable Axios's JSON parsing here](https://stackoverflow.com/a/45548588), or work with the structured data, or `useFetch` might also solve this.

Though since you're later re-parsing this into JSON, I'd just leave it as JSON here.

Reply:
Great point - thanks for the pointer. Fixed.